### PR TITLE
Adding logfile for copy errors

### PR DESCRIPTION
**Pull request by deniszh**
Add simple log file to syncer which will contain all copy errors in simple format:
`YYYY/MM/DD HH:MM:SS WHAT,METRICNAME,WHERE`
where 
WHAT - "Src" or "Dst", shows does error happened on source or destination server.
METRICNAME - metric name
WHERE - server name

Also renamed `errortolerance` flag to `error-tolerance` (to match other flags).

NB: that we're not expect massive errors during normal buckytools run, so, we can use stdlib `log` for direct logging (which is concurrently safe), w/o implementing it here by ourselves.

### CODE REVIEW

**Review by grzkv — APPROVED** [review]

File: cmd/bucky/common_sync.go [hunk 1]
```diff
@@ -281,6 +294,12 @@ func (ms *metricSyncer) countJobs(jobsd map[string]map[string][]*syncJob) int {
 	return c
 }
 
+func (ms *metricSyncer) metriclog(what string, name string, where string) {
+	if ms.metricErrLogger != nil {
+		ms.metricErrLogger.Printf("%s|%s|%s", what, name, where)
```
Comment (grzkv) [inline cmd/bucky/common_sync.go:299]: This is a rather unusual demarcation for a log and could be hard to parse. In the vast majority of cases either ":" or "," is used.

File: cmd/bucky/common_sync.go [hunk 2]
```diff
@@ -330,14 +349,18 @@ func (ms *metricSyncer) sync(jobc chan *syncJob, srcThrottling map[string]chan s
 					} else if errors.Is(err, errCantReadMetric) {
 						atomic.AddInt64(&ms.stat.copyError, 1)
 						atomic.AddInt64(&ms.stat.nodes[src].copyError, 1)
+						ms.metriclog("Src", job.oldName, src)
```
Comment (grzkv) [inline cmd/bucky/common_sync.go:352]: It is not typical to use uppercase for logs. The typical policy in Go is for all logs and error messages to be lowercase.

Comment (emadolsky) [inline cmd/bucky/common_sync.go:352]: Since "Src" & "Dst" are used in multiple lines, maybe it makes sense to get a bool as the argument or create a couple of consts to give as arguments e.g. SyncErrSrc, SyncErrDst.

Personally, I would go with the first option.

**Review by emadolsky — APPROVED** [review]
Approved. Additionally, I left a small comment which isn't a blocker.

File: cmd/bucky/common_sync.go (hunk 1 above)
Comment (deniszh) [reply cmd/bucky/common_sync.go:299]: `:` is used in date, but ',' is probably better, let's do that.

File: cmd/bucky/common_sync.go (hunk 2 above)
Comment (deniszh) [reply cmd/bucky/common_sync.go]: bool looks ok, done
